### ISSUE TITLE
Any easy way to create your own theme?

### ISSUE THREAD

**Issue by quicknir** [eclipse-color-theme/eclipse-color-theme#225]
The last time I used ECT, if you had custom settings, you didn't have a simple option to export those colors into a file and then optionally upload it to the website. Instead you had to create the theme on the website. This generally meant tweaking the color theme in your editor until you had it the way you wanted, and then copying and pasting about 50 fields. Is this still the case? Is there some convenient way to save or export a custom theme?

**Comment by ajeebkp23** [comment]
IMHO, We should have an option to import,export and modify existing themes in eclipse itself.

**Comment by paulvi** [comment]
@quicknir There is web site http://eclipsecolorthemes.org/ and Build theme button

**Comment by agentgt** [comment]
I tried to use http://eclipsecolorthemes.org and while it created a theme and supposedly an account I can't login. I didn't want to keep creating dummy accounts so I gave up. Sort of annoying because I have a theme on the web with my name on it that doesn't work that I cannot edit. 

http://eclipsecolorthemes.org/?view=theme&id=39432

should be

https://gist.github.com/agentgt/fcaf75eb8acf92e08926

**Comment by benjaminfunk** [comment]
i agree the problem with the login. i have created a theme and want to change it :(

**Comment by mebbert** [comment]
Can a developer comment on the login issues?

**Comment by fhd** [comment]
Unfortunately only @rogerdudler has access to the site, and I haven't heard from him in a while :disappointed:
